src/plugins/intel_cpu/src/plugin.h

```diff
@@ -34,6 +34,16 @@ class Plugin : public ov::IPlugin {
             "Not Implemented import_model with RemoteContext is not supported by CPU plugin!");
     };
 
+    std::shared_ptr<ov::ICompiledModel> import_model(const ov::Any& model_variant,
+                                                     const ov::AnyMap& properties) const override;
+
+    std::shared_ptr<ov::ICompiledModel> import_model(const ov::Any& model_variant,
+                                                     const ov::SoPtr<ov::IRemoteContext>& context,
+                                                     const ov::AnyMap& properties) const override {
+        OPENVINO_THROW_NOT_IMPLEMENTED(
+            "Not Implemented import_model with RemoteContext is not supported by CPU plugin!");
```

Comment:
```suggestion
            "Not Implemented import_model with RemoteContext is not supported by CPU plugin!");
```
```suggestion
            "import_model with RemoteContext is not supported by CPU plugin!");
```
`OPENVINO_THROW_NOT_IMPLEMENTED` already contains `Not implemented` message

Reply:
Done